code/game/jobs/job/job.dm

```diff
@@ -148,6 +148,31 @@
 /datum/job/proc/spawn_and_equip(var/mob/new_player/player)
 	CRASH("A job without a set spawn_and_equip proc has handle_spawn_and_equip set to TRUE!")
 
+/datum/job/proc/generate_money_account(mob/living/carbon/human/account_user)
+	account_user.job = title //TODO Why is this a mob variable at all?
```

Comment:
Doesn't look like this should be in generate_money_account

Reply:
guh